src/shipit_static_analysis/shipit_static_analysis/workflow.py

```diff
@@ -198,6 +200,7 @@ def run(self, revision):
             logger.info('Skip mozlint')
 
         logger.info('Detected {} issue(s)'.format(len(issues)))
+        stats.report_issues('all', issues)
```

Comment:
Why are you reporting the issue numbers both per-analyzer and in total? Isn't the 'in total' number just a sum?

Reply:
Exactly, but i don't know if i can do aggregation/sums directly from datadog

Reply:
We should find out

Reply:
It's possible to do simple operations in datadog interface. Removing this line.